saleor/graphql/translations/mutations.py

```diff
@@ -60,6 +62,22 @@
 }
 
 
+def validate_input_against_model(model: Model, input_data: dict):
+    errors = {}
+    for field_name, value in input_data.items():
+        model_field = model._meta.get_field(field_name)
+        if not isinstance(model_field, CharField):
+            continue
+        if value and len(value) > model_field.max_length:
+            errors[field_name] = ValidationError(
+                f"This value can't be longer than {model_field.max_length}",
+                code="too_long",
```

Comment:
Shouldn't be used `saleor/core/error_codes.py::TranslationErrorCode.TOO_LONG` here?

Reply:
I've did that originally but it resulted in mypy error that `code` should be string and not enum member. I'll see if making enum inherit both enum and str will resolve this.